src/ReaderThemeLoader.php

```diff
@@ -240,7 +240,7 @@ function ( match, startDiv ) {
 							${startDiv}
 							<# if ( data.ampActiveReaderTheme ) { #>
 								<h2 class="theme-name" id="{{ data.id }}-name">
-									<span><?php echo esc_html( _x( 'Reader:', 'prefix for theme card in list', 'amp' ) ); ?></span> {{{ data.name }}}
+									<span><?php echo esc_html( _x( 'Reader:', 'prefix for theme card in list', 'amp' ) ); ?></span> {{ data.name }}
```

Comment:
Would `data.name` need to be unescaped?

Reply:
Ah nevermind, it is being escaped now.

Reply:
WordPress core does _not_ escape this: https://github.com/WordPress/wordpress-develop/blob/e78d988aa2ee98338715520686a3bf04a1a193e7/src/wp-admin/includes/theme.php#L804

This was originally copied from core so that's why it wasn't escaped originally.

In general core has decided to not escape many things that seemingly should be (e.g. translation strings).